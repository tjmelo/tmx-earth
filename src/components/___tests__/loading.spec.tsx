import {render} from '@testing-library/react'
import '@testing-library/jest-dom'

import Loading from '../Load'

test('Should render a load message', () => {
    //do
    const childrenTest = 'TMX Earth'
    const {asFragment, getByText} = render(<Loading type={'info'} children={childrenTest} />)

    // then
    expect(getByText(childrenTest)).toBeInTheDocument()
    expect(asFragment()).toMatchSnapshot();
})