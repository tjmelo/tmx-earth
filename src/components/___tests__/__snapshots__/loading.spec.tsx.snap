// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a load message 1`] = `
<DocumentFragment>
  <div
    class="alert alert-info"
  >
    TMX Earth
  </div>
</DocumentFragment>
`;
